ngrok_extra/ngrok_extra/django/listener.py

```diff
@@ -1,10 +1,24 @@
+import json
 from django.conf import settings
 
 async def setup(listen="localhost:8000"):
     import ngrok
     # Note (james): This is where we can get settings for this app and pass them in if we want to.
-    # ngrok_config = getattr(settings, "NGROK_CONFIG", {})
     # If hot reload is on though we'll need to handle that some other way.
-    listener = await ngrok.default()
+    ngrok_config = getattr(settings, "NGROK_CONFIG", {})
+    listener = None
+    if ngrok_config:
+        listener = await listener_from_config(ngrok_config)
+    else:
+        listener = await ngrok.default()
     print(f"Forwarding to {listen} from ingress url: {listener.url()}")
     listener.forward(listen)
+
+async def listener_from_config(config):
+    import ngrok
+    session = await ngrok.SessionBuilder().authtoken_from_env().connect()
+    listener = session.http_endpoint()
+    policies = config.get("policies", {})
```

Comment:
If we add the ability to have more than just policies in the config we probably want to break out this into a helper lib.